imports/client/components/PuzzleModalForm.tsx

```diff
@@ -288,6 +330,19 @@ const PuzzleModalForm = React.forwardRef(
         </FormGroup>
       ) : null;
 
+    const allowDuplicateUrlsCheckbox =
+      !puzzle &&
+      typeof allowDuplicateUrls === "boolean" &&
+      confirmingDuplicateUrl ? (
+        <FormCheck
```

Comment:
It's good to specify a unique `id` prop to the `FormCheck` here, so that the `<label>` it ultimately produces will reference the `<input>`'s id, which makes it possible to click anywhere on the label text to toggle the checkbox instead of having to hit just the square of the checkbox.  This can be achieved in other contexts by specifying `controlId` to the containing `FormGroup`, but here we've put multiple controls into the same form group for reasonable reasons (the checkbox's meaning is bound to the url) so we should do it on the `FormCheck` itself instead.

If we're being consistent with other inputs, something like `id="jr-new-puzzle-allow-duplicate-urls"` would be perfect.

```suggestion
        <FormCheck
          id="jr-new-puzzle-allow-duplicate-urls"
```